content/projects/java-specific/collections/_index.md

```diff
@@ -9,6 +9,7 @@ prerequisites:
   - topics/java-specific/collections-and-datastructures
   soft:
   - docker/intro-to-docker
+  - docker/docker-desktop-troubleshooting
```

Comment:
This is already mentioned in intro-to-docker. Please remove the extra prerequisite